examples/grails3-hibernate5-mongodb/build.gradle

```diff
@@ -29,7 +25,6 @@ dependencies {
     implementation "org.springframework.boot:spring-boot-starter-tomcat"
     implementation "org.springframework.boot:spring-boot-starter-validation"
     console "org.grails:grails-console"
-    runtimeOnly "com.bertramlabs.plugins:asset-pipeline-grails"
```

Comment:
This one should be kept, as there are assets in the app. It's only the gradle plugin that can be removed. The gradle plugin will pre-compile assets when packaging the app. We won't do that with a test app. However the library is still needed at runtime when running tests to server assets.

Reply:
Ok, I got a little over excited to remove it completely, so that is never needed updated again in the example test projects.   I'll update this one and the PR on gorm-hibernate5.